app/src/main/res/values/strings.xml

```diff
@@ -1810,6 +1810,14 @@
     <!-- Text for the snackbar to confirm auto-close is enabled for inactive tabs -->
     <string name="inactive_tabs_auto_close_message_snackbar">Auto-close enabled</string>
 
+    <!-- Awesome bar suggestions' headers -->
+    <!-- Title for firefox suggestion -->
+    <string name="firefox_suggest_header">Firefox Suggest</string>
+    <!-- Title for suggestions when Google is default search suggestion engine. The first parameter is Google search engine. -->
+    <string name="google_search_engine_suggestion_header">%s Search</string>
```

Comment:
Nit: I wonder if the parameter is necessary for this string as it's exclusively used for Google

Reply:
@t-p-white This was a great catch. Thank you. 

You did mention on Slack that you were inspecting how other apps for this string. I was wondering how did you go about that?

Reply:
@gabrielluong in AS, in th 'root' strings file if you hover over the strings name it pops up a dialog of all the translations 🙂